.github/workflows/build.yml

```
@@ -0,0 +1,78 @@
+name: Build
+
+on: 
+  push:
+    branches:
+      - 'master'
+  pull_request:
+  workflow_dispatch:
+
+
+jobs:
+  Build:
+    name: Build appimagetool
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - ARCH: 'x86_64'
+            docker-image: 'appimagebuild'
+          - ARCH: 'i386'
+            docker-image: 'appimagebuild-i386'
+          - ARCH: 'armhf'
+            docker-image: 'appimagebuild-armhf-cross'
+          - ARCH: 'aarch64'
+            docker-image: 'appimagebuild-aarch64-cross'
+      fail-fast: false
+    env:
+      DOCKER_IMAGE: quay.io/appimage/${{ matrix.docker-image }}
+      ARCH: ${{ matrix.ARCH }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install dependencies for ARM and AARCH64
+        if: endsWith(matrix.docker-image, 'cross') 
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install binfmt-support qemu-user-static
+
+      - name: Build 
+        shell: bash
+        run: |
+          if [[ "$ARCH" == "i386" ]]; then
+            sudo dpkg --add-architecture i386
+            sudo apt-get update
+          fi
+          bash -ex ci/ci-build.sh
+
+      - name: List files
```

We could be more descriptive here:

```suggestion
      - name: List built artifacts
```

I think, however, that I'll replace these (not so great) build scripts with something based on my work in AppImageLauncher, AppImageBuild and elsewhere. They're much more robust.